person.rb

```diff
@@ -22,6 +24,10 @@ def correct_name
     @name
   end
 
+  def add_rental(date, book)
+    @rentals.push(Rental.new(date, book, self))
+  end
```

Comment:
- [ ] because you have already pushed to rentals to person in your rental class initialize method this method should look like this `Rental.new(date, book, self)`